tests/main/preseed-core20/task.yaml

```diff
@@ -0,0 +1,162 @@
+  MATCH "^etc/systemd/system/usr-lib-snapd.mount" < files.log
+  MATCH "^etc/systemd/system/snapd.spread-tests-run-mode-tweaks.service" < files.log
+  MATCH "^etc/systemd/system/dbus-fi.w1.wpa_supplicant1.service" < files.log
+  MATCH "^etc/systemd/system/snap-pc.*.mount" < files.log
+  MATCH "^etc/systemd/system/snapd.snap-repair.timer" < files.log
+  MATCH "^etc/systemd/system/sshd-keygen@.service.d/disable-sshd-keygen-if-cloud-init-active.conf" < files.log
+  MATCH "^etc/systemd/system/snapd.service.wants/usr-lib-snapd.mount" < files.log
+  MATCH "^etc/systemd/system/snapd.autoimport.service" < files.log
+  MATCH "^etc/systemd/system/final.target.wants/snapd.system-shutdown.service" < files.log
+  MATCH "^etc/systemd/system/snapd.apparmor.service" < files.log
+  MATCH "^etc/systemd/system/dbus-org.freedesktop.timesync1.service" < files.log
+  MATCH "^etc/systemd/system/sshd.service" < files.log
+  MATCH "^etc/systemd/system/snapd.socket" < files.log
+  MATCH "^etc/systemd/system/timers.target.wants/snapd.snap-repair.timer" < files.log
+  MATCH "^etc/systemd/system/dbus-org.freedesktop.resolve1.service" < files.log
+  MATCH "^etc/systemd/system/snap-core20-.*.mount" < files.log
+  MATCH "^etc/systemd/system/snap-snapd-x1.mount" < files.log
+  MATCH "^etc/systemd/system/snapd.seeded.service" < files.log
+  MATCH "^etc/systemd/system/snapd.recovery-chooser-trigger.service" < files.log
+  MATCH "^etc/systemd/system/sockets.target.wants/snapd.socket" < files.log
+  MATCH "^etc/systemd/system/cloud-init.target.wants/cloud.*" < files.log
+  MATCH "^etc/systemd/system/snapd.service" < files.log
+
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.core-fixup.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snap-pc-.*.mount" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.spread-tests-run-mode-tweaks.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snap-pc.*kernel-.*.mount" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.autoimport.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.apparmor.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snap-core20-.*.mount" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snap-snapd-.*.mount" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.seeded.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.recovery-chooser-trigger.service" < files.log
+  MATCH "^etc/systemd/system/multi-user.target.wants/snapd.service" < files.log
+  MATCH "^etc/systemd/system/getty.target.wants/" < files.log
+  MATCH "^etc/systemd/system/getty.target.wants/getty@tty1.service" < files.log
+  MATCH "^etc/systemd/system/snapd.failure.service" < files.log
+
+  MATCH "^snap/pc-kernel/current" < files.log
+  MATCH "^snap/core20/current" < files.log
+  MATCH "^snap/snapd/current" < files.log
```

Comment:
Could those be matched as:
```
MATCH "^snap/pc-kernel/current -> <rev>'  < files.log
```
if we know the revision upfront. x1 for the kernel and snapd, and some other for the base?

Reply:
Unfortunately it appears that tar list output doesn't contain symlinks (but they are restored when extracted).